capitalize.rb

```diff
@@ -0,0 +1,12 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def initialize(nameable)
+    super
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
- [ ] Kindly make sure your `correct_name()` method capitalizes the first letter of the given name and not `Uppercase` the whole name.